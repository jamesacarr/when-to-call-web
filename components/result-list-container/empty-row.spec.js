/* eslint-disable camelcase */

import React from 'react';
import { shallow } from 'enzyme';
import EmptyRow from './empty-row';

describe('<EmptyRow/>', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<EmptyRow />);
    expect(wrapper).toMatchSnapshot();
  });
});
