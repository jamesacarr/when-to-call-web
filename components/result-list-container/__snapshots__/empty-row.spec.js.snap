// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyRow/> renders correctly 1`] = `
<WithStyles(ListItem)
  dense={true}
>
  <WithStyles(ListItemText)
    primary="No Results"
  />
</WithStyles(ListItem)>
`;
